uberfire-structure/uberfire-structure-api/src/main/resources/MappingDependencies.properties

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright 2018 Red Hat, Inc. and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+JPA=org.hibernate.javax.persistence:hibernate-jpa-2.1-api:1.0.2.Final
```

Comment:
One DependencyType can have multiple values. are we going to store this with comas?

Reply:
I think it's a good time to ask everyone on bsig to map the dependencies X editors and where the dependency should be added

Reply:
thinking out loud - Could be a map, enum

Example format:
```
{
    ["project-that-uses-this-dependency-type", "dependency-type"],
    ["project-that-uses-this-dependency-type-2", "dependency-type-2"],
}
```

Reply:
@domhanak @ederign json conf like the migration file ?

Reply:
I am not sure, looks reasonable. I did not see an email about this